frontend/popup.html

```diff
@@ -17,8 +18,11 @@
       <div class="">
         <label for="search_query">Search query: </label>
         <input type="text" id="search_query" name="search_query" placeholder="How do I grow a beard?">
+        <small>This area cannot be left empty!</small>
       </div>
-      <button type="submit">Search</button>
+      <a href='' id="searchButton">Search</a>
```

Comment:
this is fixed now, it was edited, so that I could use an href attribute, but I fixed this with JavaScript